src/enkf/gridinfo_fv3reg.f90

```diff
@@ -67,7 +68,8 @@ module gridinfo
 integer,public :: npts
 integer,public :: ntrunc
 ! supported variable names in anavinfo
-character(len=max_varname_length),public, dimension(8) :: vars3d_supported = (/'u   ', 'v   ', 't ', 'q   ', 'oz  ', 'cw  ', 'tsen', 'prse' /)
+character(len=max_varname_length),public, dimension(15) :: vars3d_supported = (/'u   ', 'v   ', 'w ', 't ', 'q   ', 'oz  ', 'cw  ', 'tsen', 'prse', &
```

Comment:
I think the part related to direct dbz DA is already in EMC GSI master.  It may need some manual treatment when trying to merge back to EMC GSI